### DISCUSSION TITLE
V2 not pulling through devices/entities

### DISCUSSION THREAD

**Discussion by Shellers44** [tuya/tuya-home-assistant#366] [Q&A]
I've just installed v2 but there is no 'Configure' click button or any devices or entities listed on the v2 integration in HA. I've made sure that the cloud project is set up with my Smart Life account on Tuya IOT. I've tried restarting numerous times, cleared the cache etc but I cannot get my devices in the new integration. Any ideas what could be wrong here? I did select PAAS Smart Home option when setting up so there's nothing obvious that I can see, but I am fairly new to HA etc. TIA

**Comment by Shellers44** [comment]
Answered my own issue, I needed to go into 'Smart Home Devices Management' in Tuya IOT to authorise access to the project. Not sure why I had to do this step, I didn't when I tried setting up Tuya local. Anyway, it's fixed!

**Reply by Keltharak** [reply]
Thanks ! I had the same problem and you gave me the solution. 

Too bad the documentation is not up to date.

**Reply by navigat0** [reply]
Sadly I do not see  Smart Home Devices Management API to choose from.  I do see all my devices listed in Tuya IOT but they do not show up through the hass integration.

**Reply by jeffoberholtzer** [reply]
I am having the same issue, I don't see any devices once the Tuya integration is installed

**Reply by Trukmeister** [reply]
I am having the same issue, I don't see any devices once the Tuya integration is installed and authenticated.
